integration_tests.py

```diff
@@ -79,6 +79,23 @@ def test(self):
             self.assertEqual(query.status, 'SUCCESS')
             self.assertListOfDictsEqual(query.to_dict(), expected)
 
+class KillQueryTest(MyriaTestBase):
+    def test(self):
+        # This is a long-running program
+        program = """
+x = [0 as exp, 1 as val];
+do
+  x = [from x emit exp+1 as exp, 2*val as val];
+while [from x emit max(exp) < 1000];
+store(x, powersOfTwo);
+"""
+        plan = self.connection.compile_program(program)
+        # We submit with submit_plan so as to not block
+        query = MyriaQuery.submit_plan(plan)
```

Comment:
It would be nice to just call a nonblocking `MyriaQuery` API with the program string. I guess this could be done in `myria-python` by either adding a new method to `MyriaQuery` that just generates the plan and calls `MyriaConnection.submit_query()`, or by adding a nonblocking version of `MyriaConnection.execute_program()`. @BrandonHaynes could you weigh in?